server/index.js

```diff
@@ -8,7 +8,7 @@ const staticMiddleware = require('./static-middleware');
 const sessionMiddleware = require('./session-middleware');
 
 const app = express();
-
+let sessionCartId = null;
```

Comment:
This value should not be global, otherwise _every_ user will share the same cart.